cmswww/database/cockroachdb/cockroachdb.go

```diff
@@ -415,5 +416,9 @@ func New(dataDir, dbName, username, host string) (*cockroachdb, error) {
 		&InvoicePayment{},
 	)
 
+	// Add foreign key constraints
```

Comment:
They're really indexes, not foreign key constraints, right?

Reply:
Yep, I had some foreign key stuff in there too (which the indexes were required for), but I removed the foreign key stuff as I was having a warning and thought I'd bundled enough in this PR already.  I will update the comment.